### ISSUE TITLE
Can't save query to table dbplyr

### ISSUE THREAD

**Issue by mikaelhogerman** [oxford-pharmacoepi/MegaStudy#69]
Hi,

I tried to re-run IncidencePrevalence and now I get this error that occures in line 48 (generateDrugUtilisationCohortSet)
![Näyttökuva (27)](https://github.com/oxford-pharmacoepi/MegaStudy/assets/148959111/46306dc6-eafc-4fe7-a4e2-39cbc22743fe)

**Comment by tiozab** [comment]
@catalamarti any idea? this DP gets this problem a lot with column [xz] has type "unknown"

**Comment by edward-burn** [comment]
Hi @mikaelhogerman, can I check what version of postgres you are using?

(@tiozab afraid I haven't seen this one before.....)

**Comment by mikaelhogerman** [comment]
@edward-burn
PostgreSQL 9.5.25 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609, 64-bit

**Comment by tiozab** [comment]
@mikaelhogerman have you gotten over this issue?

**Comment by mikaelhogerman** [comment]
@tiozab No I haven't. I'm not sure if this can affect DUS, but this issue was for IncidencePrevalence.

**Comment by tiozab** [comment]
but we have received your incidence prevalence results ? 
![image](https://github.com/user-attachments/assets/f93c034d-724b-4045-8b83-57e2dadd3c86)

**Comment by mikaelhogerman** [comment]
Yes I was able to run IncidencePrevalence with old github version after some modifications. But when I tried to run DUS and got error I loaded all the newest git versions (including IncidencePrevalence) and that time it gave me this error for IncidencePrevalence.

Issue was that our results gave zeros for denominator_days_prior_observation like in git error #42.
And that was the reason we wanted to run it again but then it failed and gave this error.
